src/pages/resources/migration/android/index.md

```diff
@@ -0,0 +1,7 @@
+# Migration guides for Adobe Experience Platform Android SDKs
+
+This document provides a collection of resources to help you migrate between various versions of the Adobe Experience Platform Android SDKs.
+
+## Migrate to Adobe Experience Platform 2.x Android SDKs
```

Comment:
This comment applies for all files and occurrences in this PR:
- to eliminate any confusion with Android/iOS versions, can we use this consistent naming across our docs:
Adobe Experience Platform 2.x SDKs for Android or Adobe Experience Platform 4.x SDKs for iOS.
- you may use the accepted convention for simplifying names in second/third occurrence in the same file:
Experience Platform Mobile SDKs / Platform Mobile SDKs / Mobile SDKs